nest/command_line.py

```diff
@@ -148,12 +148,14 @@ def get_device(napi, args, structure):
         return structure.cameras[args.index]
 
 
-def handle_camera_show(device):
-    print('Device                : %s' % device.name)
-    # print('Model               : %s' % device.model) # Doesn't seem to work
-    print('Serial                : %s' % device.serial)
-    print('Where                 : %s' % device.where)
-    print('Where ID              : %s' % device.where_id)
+def handle_camera_show(device, count=0):
```

Comment:
If this is not directly related to the streaming API, please submit changes into a pull request in the main repo.

Reply:
I just want to include a showcase the usage of update_event and the stream API. I added a new argument to keep the command line tool behavior backward compatible. Does it make sense now?

Reply:
Thanks. This should work.